src/store/place/restaurants.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
convert to json into ts it self, and I suggest preprecess this chunk in this file region.

Reply:
Again, I think it should be converted into script file and in that file, all preprocessing should be occurred.